<launch>
  <arg name="config_path" default="$(find point_cloud_projection)/configs/config.yaml"/>

  <node pkg="point_cloud_projection"
        type="ColoriseMap"
        name="colorise_map_node"
        output="screen">
    <param name="config_path" value="$(arg config_path)"/>
    <param name="save_pcd_path" value="$(find point_cloud_projection)/output/colored_map.pcd"/>
    <param name="map_topic" value="/lio_sam/mapping/cloud_registered"/>
    <param name="odom_topic" value="/lio_sam/mapping/path"/>
    <param name="min_color_frames" value="3"/>
  </node>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find point_cloud_projection)/configs/vis.rviz">
    <param name="output" value="screen"/>
  </node>
</launch>
